content/changelog/2024-09-13.md

```diff
@@ -0,0 +1,32 @@
+---
+description: tbd
+---
+
+### This week’s changelog – Live from Lisbon!
+
+![Neon Team Portugal](/docs/relnotes/neon_team.jpg)
+
+Our team is in Lisbon this week, wrapping up Q3 and refining our roadmap. It's been a great opportunity to reconnect, welcome new colleagues, meet with customers, and focus on executing our plans.
+
+As we look ahead as a team, here's a quick look back at some key features we've released over the past few months:
+
+- **Inbound Logical Replication**: Enables low-downtime migrations to Neon, making it easy to use Neon as a development environment with features like branching and [GitHub integration](/docs/guides/neon-github-integration). It also supports migrations from other providers and between Neon projects. Check out our guides on [Replicating Data to Neon](/docs/guides/logical-replication-guide#replicate-data-to-neon) and [Project-to-Project Replication](/docs/guides/logical-replication-neon-to-neon).
+- **Organizations**: Now available for Early Access users. Create and manage your organization's projects and teams with ease. Join our [Early Access Program](https://console.neon.tech/app/settings/early-access) and refer to the [Organizations docs](/docs/manage/organizations) for more information.
+- **Autoscaling GA**: Neon's Autoscaling feature is now generally available, automatically adjusting resources as needed. Read the announcement [here](https://neon.tech/blog/neon-autoscaling-is-generally-available) and explore the [Autoscaling docs](/docs/introduction/autoscaling).
+- **Autoscaling on Free Plan**: Free Plan users can now experience Neon's Autoscaling, within reasonable limits. Learn more in the [Autoscaling Guide](/docs/guides/autoscaling-guide).
+- **A New Business Plan**: Offers 500 GiB of storage and 1,000 compute hours, with potential cost savings for customers needing more storage than offered by our Scale plan. Learn more on our [Pricing](https://neon.tech/pricing) and [Plans](/docs/introduction/plans) pages.
+
+For a peek at what we'll be delivering next, check out the [Neon roadmap](/docs/introduction/roadmap). You can expect to see some additions here as soon as we finish fine-tuning our Q4 plan. Stay tuned!
```

Comment:
Maybe remove this paragraph about the roadmap entirely?

Reply:
removed.